app/assets/stylesheets/webapp/form.scss

```diff
@@ -1,4 +1,5 @@
 .controller-ncr-work_orders,
+.controller-gsa18f-procurements,
 .controller-feedback {
```

Comment:
Maybe we should just make a custom class for everything we want this applied to? Or is it every `<form>`? Feel free to make a story if you feel it's outside of the scope of this story.